packages/api/src/command/webhook/fitbit.ts

```diff
@@ -199,9 +208,9 @@ async function createAndSendCustomerPayloads(dataByCustomer: Dictionary<Entry[]>
           dataAndUserList.map(du => du.userId)
         );
       } catch (err) {
-        log(`Error on processData: ${err}`);
+        log(`Failed to create and send customer payloads: ${err}`);
         capture.error(err, {
-          extra: { context: `webhook.processData.fitbit`, err },
+          extra: { context: `fitbit.webhook.createAndSendCustomerPayloads`, err },
```

Comment:
NABD, but I thought we had the provider as suffix on the context, instead of prefix?

Reply:
Good catch. I'll make this consistent with other providers.